samples/suppliers.rego

```diff
@@ -0,0 +1,9 @@
+package suppliers
+
+default allow = false
+
+allow {
+    input.method == "query"
+    input.did != null
+    input.did == input.suppliers[_]
```

Comment:
so this just says that the input did needs to be in the array of input suppliers?

Reply:
yep, tried to simplify it down as much as possible. To the point where it's somewhat arbitrary